packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -232,6 +232,13 @@ export type CallCompositeOptions = {
    * Options for controlling video tile.
    */
   videoTilesOptions?: VideoTilesOptions;
+  /**
+   * Whether to auto show the DTMF Dialer when the call starts in supported scenarios.
+   * - Teams Voice Application like Call queue or Auto Attendant
+   * - PSTN Calls
+   * @defaultValue true
```

Comment:
Azure guidance is to avoid defaults of true. Instead rename to the negative: `disableAutoShowDtmfDialer`, or (per Azure other guidance) avoid boolean:
```
dialpadAutoShowBehavior: 'none' | 'detectedVoiceApplications'
```